app/test_split_integer.py

```diff
@@ -2,20 +2,35 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    value = 15
+    number_of_parts = 5
+    result = split_integer(value, number_of_parts)
+    assert sum(result) == value
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    value = 20
+    number_of_parts = 4
+    result = split_integer(value, number_of_parts)
+    assert all(part == value // number_of_parts for part in result)
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    value = 10
+    number_of_parts = 1
+    result = split_integer(value, number_of_parts)
+    assert result == [value]
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal() -> None:
-    pass
+    value = 25
+    number_of_parts = 5
+    result = split_integer(value, number_of_parts)
+    assert result == sorted(result)
 
 
 def test_should_add_zeros_when_value_is_less_than_number_of_parts() -> None:
-    pass
+    value = 3
+    number_of_parts = 5
+    result = split_integer(value, number_of_parts)
+    assert result == [0, 0, 1, 1, 1]
```

Comment:
Rewrite tests so that there would be only an `assert` statement inside of each test without allocation of additional variables to be more concise.